numpy/core/tests/test_indexerrors.py

```diff
@@ -1,5 +1,8 @@
 import numpy as np
-from numpy.testing import assert_raises
+from numpy.testing import (
+        assert_raises, assert_raises_regex
```

Comment:
Just nitpicking (don't bother, I constantly ignore it), but I think if the closing bracket is on the next line, it is good style to end with a comma.

Reply:
Thanks for your advice! I'm really new to Numpy. I'd love to accept this kind of advice to keep the same code style with other code!

Reply:
The idea behind ending the line with a comma is that it allows additional modification in the future with minimal disruption to the code